lib/session/freshman_session.dart

```diff
@@ -16,19 +16,17 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import 'package:kite/credential/symbol.dart';
 import 'package:kite/network/session.dart';
-import 'package:kite/storage/init.dart';
 import 'package:kite/util/logger.dart';
 
-class FreshmanSession extends ISession {
+class FreshmanSession {
```

Comment:
为啥不要 `ISession` 了？ @zhangzqs @liplum

Reply:
Because the `Auth.freshmanCredential` can't be passed through into `request(...)` in ISession.
In [olde code](https://github.com/SIT-kite/kite-app/blob/master/lib/module/freshman/page/login.dart#L81), the FreshmanCredential is even saved into `Kv.freshman.freshmanAccount` temporarily for this disabled ISession interface.

But in [new code](https://github.com/SIT-kite/kite-app/blob/new-auth/lib/credential/delegate.dart#L44), we cannot do that, because It leads to an unexcepted `lastFreshmanAuthTime` change due to a prompt from users.